opba-db/src/main/java/de/adorsys/opba/db/domain/entity/Payment.java

```diff
@@ -85,6 +85,9 @@
     @LastModifiedDate
     private Instant modifiedAt;
 
+    @Column(name = "image_data")
```

Comment:
This data must be in context, not in payment

Reply:
Removed